cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -268,6 +268,17 @@ cros_gralloc_driver::cros_gralloc_driver()
 			DRV_INIT(drv_video_, gpu_grp_type, video_idx)
 		if ((virtio_node_idx != -1) && (virtio_node_idx != renderer_idx))
 			DRV_INIT(drv_kms_, gpu_grp_type, virtio_node_idx)
+		if (drv_kms_) {
```

Comment:
drv_kms_ can i915 node, then it doesn't make sense to call i915 node with drv_query_dev_feature

Reply:
actually in the drv.c
bool drv_query_dev_feature(struct driver * drv)
{
        bool ret = false;
        assert(drv);
        assert(drv->backend);

        if (drv->backend->query_dev_feature) {
                ret = drv->backend->query_dev_feature();
        }
        return ret;
}
if driver do not have the backend query_dev_feature implemnt. it will return false.
Anyway i will modify the if condition.